.github/workflows/work_precision_sets.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: python -m pip install pybamm==${{ env.VERSION }}
       - name: Run time_vs_* benchmarks for PyBaMM v${{ env.VERSION }}
         run: |
-          python benchmarks/work_precision_sets/time_vs_dt_max.py
+          python benchmarks/work_precision_sets/time_vs_dt_event.py
```

Comment:
This file also needs to be renamed.

Reply:
Done in [`68c2925`](https://github.com/pybamm-team/PyBaMM/pull/3106/commits/68c29256797fd20fdef007f54264a1a48925d493)